demos/mantle_convection/adaptive_base_case/adaptive_base_case.py

```diff
@@ -0,0 +1,362 @@
+# Mantle convection base case using adaptive meshing
+# ==================================================
+# This tutorial demonstrates the adaptive meshing capabality available in
+# G-ADOPT, which dynamically adapts the mesh to focus resolution where needed.
+# As demonstrated in [Davies et al.
+# (2011)](https://doi.org/10.1029/2011GC003551) this may significantly reduce
+# the computational requirements of mantle convection models, in particular
+# when using anisotropic metric-based adaptivity which can very efficiently
+# resolve the anisotropic features in geodynamical flows.
```

Comment:
@drhodrid I let you judge here if you would like additional information or if this is sufficient.